releases/release-1.23/release-notes/maps/pr-105405-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 105405
+releasenote:
+  text: Ephemeral containers have reached beta maturity and are now available by default.
```

Comment:
Change to: Ephemeral containers graduated to beta and are now available by default.